pkgs/data/misc/mobile-broadband-provider-info/default.nix

```diff
@@ -1,17 +1,29 @@
-{ lib, stdenv, fetchurl, gnome, libxslt }:
+{ lib
+, stdenv
+, fetchurl
+, gnome
+, libxslt
+, libxml2
+, meson
+, ninja
+, pkg-config
+}:
 
 stdenv.mkDerivation rec {
   pname = "mobile-broadband-provider-info";
-  version = "20230416";
+  version = "20240407";
 
   src = fetchurl {
     url = "mirror://gnome/sources/${pname}/${version}/${pname}-${version}.tar.xz";
-    sha256 = "sha256-4+FAqi23abKZ0d+GqJDpSKuZ1NOIMTAsRS0ft/hWiuw=";
+    hash = "sha256-ib/v8hX0v/jpw/8uwlJQ/bCA0R6b+lnG/HGYKsAcgUo=";
   };
 
   nativeBuildInputs = [
-    # fixes configure: error: xsltproc not found
-    libxslt
+    libxslt # for xsltproc
+    libxml2 # for xmllint
+    meson
+    ninja
+    pkg-config
```

Comment:
pkg-config is not actually used. Meson just checks for it pre-emptively.

Reply:
Good point! Dropped as:

```diff
--- a/pkgs/data/misc/mobile-broadband-provider-info/default.nix
+++ b/pkgs/data/misc/mobile-broadband-provider-info/default.nix
@@ -23,7 +23,6 @@ stdenv.mkDerivation rec {
     libxml2 # for xmllint
     meson
     ninja
-    pkg-config
   ];

   passthru = {
```

Still builds.